src/lib/connectors/idm.js

```diff
@@ -35,6 +36,35 @@ usersClient.getUserByUserName = async userName => {
   return head(data);
 };
 
+/**
+ * call IDM to check user credentials and create email change record
+ */
+const createEmailChangeRecord = (userId, password) => {
+  const url = `${process.env.IDM_URI}/user/change-email-address/start`;
```

Comment:
For all the URLs can we use:
```
config.services.idm 
config.services.crm
```
etc.
Rather than using the environment variables directly